<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ladefahrrad</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Desktop/favicon.ico</normaloff>../../Desktop/favicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTreeWidget" name="treeView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <column>
           <property name="text">
            <string>Typ</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Wert</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Ändern</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Spannung:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="voltageSlider">
                 <property name="maximum">
                  <number>10000</number>
                 </property>
                 <property name="singleStep">
                  <number>100</number>
                 </property>
                 <property name="pageStep">
                  <number>100</number>
                 </property>
                 <property name="tracking">
                  <bool>true</bool>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksBelow</enum>
                 </property>
                 <property name="tickInterval">
                  <number>200</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Strom:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="currentSlider">
                 <property name="maximum">
                  <number>10000</number>
                 </property>
                 <property name="singleStep">
                  <number>100</number>
                 </property>
                 <property name="pageStep">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksBelow</enum>
                 </property>
                 <property name="tickInterval">
                  <number>200</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="automodeCheckbox">
                 <property name="text">
                  <string>Automode</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="lamp1Checkbox">
                 <property name="text">
                  <string>Lampe 1</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="lamp2Checkbox">
                 <property name="text">
                  <string>Lampe 2</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="lamp3Checkbox">
                 <property name="text">
                  <string>Lampe 3</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="lamp4Checkbox">
                 <property name="text">
                  <string>Lampe 4</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Log</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QTextEdit" name="logView">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                 <property name="html">
                  <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Oxygen-Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="holdButton">
                   <property name="text">
                    <string>Anhalten</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="clearButton">
                   <property name="text">
                    <string>Löschen</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Debug</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Nachricht:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="idChooser">
                 <item>
                  <property name="text">
                   <string>g01</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>g02</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>g03</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>g04</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="messageInput">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="maxLength">
                  <number>5</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="sendButton">
                 <property name="text">
                  <string>Senden</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Demo</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Demo auswählen:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QListWidget" name="listWidget">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <item>
                  <property name="text">
                   <string>Modus 1</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Modus 2</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Modus 3</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Modus 4</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="demo_s">
                 <property name="text">
                  <string>Demo starten</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDemo"/>
   <addaction name="actionDemoBeenden"/>
   <addaction name="separator"/>
   <addaction name="actionEinstellungen"/>
   <addaction name="separator"/>
   <addaction name="resetActionButton"/>
   <addaction name="separator"/>
   <addaction name="actionHilfe"/>
  </widget>
  <action name="actionEinstellungen">
   <property name="icon">
    <iconset>
     <normaloff>../icons/network-disconnect.svg</normaloff>../icons/network-disconnect.svg</iconset>
   </property>
   <property name="text">
    <string>Einstellungen</string>
   </property>
  </action>
  <action name="actionDemoBeenden">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/media-playback-stop.svg</normaloff>../icons/media-playback-stop.svg</iconset>
   </property>
   <property name="text">
    <string>Demo Beenden</string>
   </property>
   <property name="toolTip">
    <string>Demo Beenden</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_ber">
   <property name="text">
    <string>Über..</string>
   </property>
  </action>
  <action name="actionDemo">
   <property name="icon">
    <iconset>
     <normaloff>../icons/media-playback-start.svg</normaloff>../icons/media-playback-start.svg</iconset>
   </property>
   <property name="text">
    <string>Demo</string>
   </property>
  </action>
  <action name="actionSenden">
   <property name="icon">
    <iconset>
     <normaloff>../icons/mail-send.svg</normaloff>../icons/mail-send.svg</iconset>
   </property>
   <property name="text">
    <string>Senden</string>
   </property>
  </action>
  <action name="actionHilfe">
   <property name="icon">
    <iconset>
     <normaloff>../icons/system-help.svg</normaloff>../icons/system-help.svg</iconset>
   </property>
   <property name="text">
    <string>Hilfe</string>
   </property>
  </action>
  <action name="actionWerte_ndern">
   <property name="icon">
    <iconset>
     <normaloff>../icons/document-edit.svg</normaloff>../icons/document-edit.svg</iconset>
   </property>
   <property name="text">
    <string>Werte ändern</string>
   </property>
  </action>
  <action name="actionLampe_Senden">
   <property name="icon">
    <iconset>
     <normaloff>../icons/games-hint.svg</normaloff>../icons/games-hint.svg</iconset>
   </property>
   <property name="text">
    <string>Lampe Senden</string>
   </property>
  </action>
  <action name="resetActionButton">
   <property name="icon">
    <iconset>
     <normaloff>../icons/appointment-new.svg</normaloff>../icons/appointment-new.svg</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
